webapp-django/crashstats/settings/base.py

```diff
@@ -643,11 +644,22 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 # then set this to False
 SESSION_COOKIE_SECURE = config('SESSION_COOKIE_SECURE', True, cast=bool)
 
-# To get your Sentry key, go to https://errormill.mozilla.org/
-RAVEN_CONFIG = {
-    'dsn': config('RAVEN_DSN', '')  # see https://errormill.mozilla.org/
-}
-
+# When socorro is installed (python setup.py install), it will create
+# a file in site-packages for socorro called "socorro/socorro_revision.txt".
+# If this socorro was installed like that, let's pick it up and use it.
+try:
+    SOCORRO_REVISION = resource_string('socorro', 'socorro_revision.txt')
+except IOError:
+    SOCORRO_REVISION = None
```

Comment:
A cool thing about this is that we no longer need to depend on the middleware doing this. 
And things like whatsdeployed.io should be able to get to this much quicker than having to execute [this beast](https://github.com/mozilla/socorro/blob/085f61878a4d6ad2384e1314e13a875ed666447e/socorro/external/postgresql/server_status.py#L19-L86) every time. 

The `except IOError` is for local devs.